frontend/src/components/Loader/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Spinner } from 'react-bootstrap';
+
+interface IProps {
+  isLoading: boolean;
+  height?: string;
+}
+
+const Loader: React.FC<IProps> = ({
+  isLoading,
+  children,
+  height = '100vh'
+}) => (
+  isLoading
+    ? (
+      <div
+        className="d-flex justify-content-center align-items-center"
+        style={{
+          position: 'relative', height: `${height}`
```

Comment:
Let's make the Loader component have 100% height of the parent and position absolute. Then we won't need to set height explicitly each time we want to wrap our components into Loader, all we will need to do is to set `position: relative` to the proper parent, and then the Loader height will adjust automatically